clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -165,12 +180,34 @@ class ClioWebClient private[client] (
     ).via(parserFlow)
   }
 
-  def query[I](clioIndex: ClioWebClient.QueryAux[I])(
+  def preformattedQuery[I](clioIndex: ClioWebClient.QueryAux[I])(
```

Comment:
Hrm, not sure if "preformatted" is the right description... maybe "simple"? Since the server re-formats it.